### PR TITLE
update changelog for v0.8.0

### PR DESCRIPTION
**Pull request by skriss**
Signed-off-by: Steve Kriss <steve@heptio.com>

Add changelog entry for v0.8.0 along with upgrade instructions from v0.7.1

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -27,11 +28,43 @@ Bug Fixes / Other Changes:
...
+    * [AWS](https://github.com/heptio/ark/blob/v0.8.0/docs/aws-config.md#credentials-and-configuration)
+    * [Azure](https://github.com/heptio/ark/blob/v0.8.0/docs/azure-config.md#credentials-and-configuration)
+    * [GCP](https://github.com/heptio/ark/blob/v0.8.0/docs/gcp-config.md#credentials-and-configuration)
+
+    When you get to the secret creation step, if you don't have your `credentials-ark` file handy, 
+    you can copy the existing secret from your `heptio-ark-server` namespace into the `heptio-ark` namespace:
+    ```bash
+    kubectl get secret/cloud-credentials -n heptio-ark-server --export -o json | \
+      jq '.metadata.namespace="heptio-ark"' | \
+      kc apply -f -
```
Comment (nrb) [inline CHANGELOG.md]: Is `kc` an alias for `kubectl`?

File: CHANGELOG.md [hunk 2]
```diff
@@ -27,11 +28,43 @@ Bug Fixes / Other Changes:
...
+    ```
+
+2. You can now safely delete the `heptio-ark-server` namespace:
+    ```bash
+    kubectl delete namespace heptio-ark-server
+    ```
+
+3. Run the **Start the server** commands for your cloud:
```
Comment (nrb) [inline CHANGELOG.md]: I re-applied the prereqs files with my [upgrade script](https://gist.github.com/nrb/815adcca7e9964c3e28c1523d07c57b7). Diffing it from v0.7.1, looks like some of that changed namespaces (which could be moved with a `jq` oneliner`), and we added the `DeleteBackupRequest` CRD.

```
diff --git c/examples/common/00-prereqs.yaml w/examples/common/00-prereqs.yaml
index 843e043..67be03f 100644
--- c/examples/common/00-prereqs.yaml
+++ w/examples/common/00-prereqs.yaml
@@ -88,23 +88,32 @@ spec:
     kind: DownloadRequest
 
 ---
-apiVersion: v1
-kind: Namespace
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
 metadata:
-  name: heptio-ark
+  name: deletebackuprequests.ark.heptio.com
+  labels:
+    component: ark
+spec:
+  group: ark.heptio.com
+  version: v1
+  scope: Namespaced
+  names:
+    plural: deletebackuprequests
+    kind: DeleteBackupRequest
 
 ---
 apiVersion: v1
 kind: Namespace
 metadata:
-  name: heptio-ark-server
+  name: heptio-ark
 
 ---
 apiVersion: v1
 kind: ServiceAccount
 metadata:
   name: ark
-  namespace: heptio-ark-server
+  namespace: heptio-ark
   labels:
     component: ark
 
@@ -117,7 +126,7 @@ metadata:
     component: ark
 subjects:
   - kind: ServiceAccount
-    namespace: heptio-ark-server
+    namespace: heptio-ark
     name: ark
 roleRef:
   kind: ClusterRole
```

**Comment by skriss** [comment]
Fixed `kc` in changelog. Rejiggered roadmap.

File: CHANGELOG.md [hunk 3]
```diff
@@ -27,11 +28,43 @@ Bug Fixes / Other Changes:
...
+
+2. You can now safely delete the `heptio-ark-server` namespace:
+    ```bash
+    kubectl delete namespace heptio-ark-server
+    ```
+
+3. Execute the commands from the **Start the server** section for your cloud:
+    * [AWS](https://github.com/heptio/ark/blob/v0.8.0/docs/aws-config.md#start-the-server)
+    * [Azure](https://github.com/heptio/ark/blob/v0.8.0/docs/azure-config.md#start-the-server)
+    * [GCP](https://github.com/heptio/ark/blob/v0.8.0/docs/gcp-config.md#start-the-server)
```
Comment (Bradamant3) [inline CHANGELOG.md]: In the preceding lines, you're linking explicitly to the markdown files in the GH repository. I can construct the URL for the equivalent links to the docs site, which might be better. We should also probably get these steps into the docs very soon -- maybe next week after other releases are out the door? Easy enough to rebuild for the same version.

Comment (nrb) [reply CHANGELOG.md]: :+1: to using the docs site URL vs repo files.

Comment (skriss) [reply CHANGELOG.md]: Good point, will do.

**Comment by skriss** [comment]
updated links.

**Comment by skriss** [comment]
@ncdc any comments before we merge?

File: ROADMAP.md [hunk 4]
```diff
@@ -1,50 +1,40 @@
...
-- Always try to upload backup log file, even if backup fails - [#250](https://github.com/heptio/ark/issues/250)
-- Support pre and post pod exec hooks - [#244](https://github.com/heptio/ark/issues/244)
-- Add serviceaccounts to list of default prioritized resources during restore - [#239](https://github.com/heptio/ark/issues/239)
-
 ## Upcoming Versions
 
 The following versions, dates, and features are approximate and are subject to change.
 
-### v0.8.0 - ~ 2018-04-19
+### v0.9.0 - ~ 2018-06-14
```
Comment (nrb) [inline ROADMAP.md]: I don't think this should be a blocker for v0.8.0, but we should probably take the items that were formerly v0.9.0 and move them to v0.10.0; the list referenced here looks kinda big.

Comment (skriss) [reply ROADMAP.md]: makes sense - we should revisit that next week.

**Comment by skriss** [comment]
Alright, just ran through another test on a fresh AWS cluster for the 0.7.1->0.8.0 upgrade, looks good. @nrb can you merge if lgty?

**Pull request merged**